### DISCUSSION TITLE
Apply LR finder from pytorch lightning if `learning_rate` cannot be pre-determined

### DISCUSSION THREAD

**Discussion by carusyte** [Nixtla/neuralforecast#1262] [Ideas]
Wondering if we could embed this feature which is especially useful for complex models and improve performance. Apply lr_finder if `learning_rate` is not given to the model, and provide means to retrieve optimum learning rate after `fit()`.

Ref: https://lightning.ai/docs/pytorch/stable/advanced/training_tricks.html#learning-rate-finder
